app/wama.web/Views/AdminConsole/UserTool/CreateNewUserAccount.cshtml

```diff
@@ -44,6 +44,15 @@
                 <label asp-for="InstitutionAffiliation"></label>
                 <select asp-for="InstitutionAffiliation" asp-items="@Html.GetEnumSelectList<InstitutionAffiliation>()" class="form-control"></select>
             </div>
+                <div class="form-group col-xs-12">
+                <label asp-for="AccountType"></label>
+                <select asp-for="AccountType" 
+                        asp-items="@Html.GetEnumSelectList<UserAccountType>()" 
+                        class="form-control" 
+                        >
+                    <option selected>@ViewBag.AccountType</option>
```

Comment:
have you check comments in fix but i dont like?
are there any grammer for tag helper select to set default value with enum?
i couldnt find it on microsoft documentation, it tell me the way with no enum (use bunch of option select)
but does not tell me how to do with enum, any idea for fix?